src/Testcontainers/Configurations/Modules/Databases/AzuriteTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,279 @@
+namespace DotNet.Testcontainers.Configurations.Modules.Databases
+{
+  using System;
+  using System.Collections.Generic;
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  public class AzuriteTestcontainerConfiguration : IDisposable
+  {
+    // TODO: Add support for these options based on request
+    // azurite.cert Path to a PEM or PFX cert file.Required by HTTPS mode.
+    // azurite.key Path to a PEM key file. Required when azurite.cert points to a PEM file.
+    // azurite.pwd PFX cert password. Required when azurite.cert points to a PFX file.
+    // azurite.oauth OAuth authentication level. Candidate level values: basic.
+    // AZURITE_ACCOUNTS environment variable to set account names and keys.
+
+    /// <summary>
+    /// Default Blob service listening port. Default is 10000.
+    /// </summary>
+    [PublicAPI]
+    public const int DefaultBlobPort = 10000;
+
+    /// <summary>
+    /// Default Queue service listening port. Default is 10001.
+    /// </summary>
+    [PublicAPI]
+    public const int DefaultQueuePort = 10001;
+
+    /// <summary>
+    /// Default Table service listening port. Default is 10002.
+    /// </summary>
+    [PublicAPI]
+    public const int DefaultTablePort = 10002;
+
+    /// <summary>
+    /// Default Workspace location folder path. Default is /data.
+    /// </summary>
+    [PublicAPI]
+    public const string DefaultLocation = "/data";
+
+    internal const string DefaultBlobEndpoint = "0.0.0.0";
+    internal const string DefaultQueueEndpoint = "0.0.0.0";
+    internal const string DefaultTableEndpoint = "0.0.0.0";
+
+    private const string DefaultAzuriteImage = "mcr.microsoft.com/azure-storage/azurite:3.18.0";
+
+    private bool runBlobOnly;
+    private bool runQueueOnly;
+    private bool runTableOnly;
+    private string location;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzuriteTestcontainerConfiguration" /> class with default Azurite image.
+    /// </summary>
+    /// <param name="image">The Docker image.</param>
+    public AzuriteTestcontainerConfiguration()
+      : this(DefaultAzuriteImage)
+    {
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzuriteTestcontainerConfiguration" /> class.
+    /// </summary>
+    /// <param name="image">The Docker image.</param>
+    protected AzuriteTestcontainerConfiguration(string image)
+    {
+      this.Image = image;
+      this.Environments = new Dictionary<string, string>();
+      this.OutputConsumer = Consume.DoNotConsumeStdoutAndStderr();
```

Comment:
You don't need (and use) that, that's the default value. Developers can still set the output consumer with their builder configuration.

Reply:
Removed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)